src/main/resources/view/PersonDetails.fxml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
```

Comment:
I think wildcard imports should be avoided in this case as well, let's remember to address this in the future.